src/modules/logicModule/LM_PC_PaymentRouter_v1.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+// Internal Interfaces
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {IAuthorizer_v1} from "@aut/IAuthorizer_v1.sol";
+import {ILM_PC_PaymentRouter_v1} from
+    "@lm/interfaces/ILM_PC_PaymentRouter_v1.sol";
+import {
+    IERC20PaymentClientBase_v1,
+    IPaymentProcessor_v1
+} from "@lm/abstracts/ERC20PaymentClientBase_v1.sol";
+
+// Internal Dependencies
+import {
+    ERC20PaymentClientBase_v1,
+    Module_v1
+} from "@lm/abstracts/ERC20PaymentClientBase_v1.sol";
+
+// Internal Libraries
+
+// External Libraries
+
+// NOTE: Development halted until the payment processor can handle multiple tokens,  and not only the native orchestrator one
+
+/**
+ * @title   Payment Router
+ *
+ * @notice  Tis module is a stopgap solution to enable pushing payments to the Payment Processor.
```

Comment:
Also I think this is a valid Module in General
I wouldnt call it stopgap solution

Reply:
Stopgap because we are gonna redesign the whole thing anyway. But I'll remove it for now